packages/google_maps_flutter/example/lib/map_ui.dart

```diff
@@ -183,6 +186,36 @@ class MapUiBodyState extends State<MapUiBody> {
     );
   }
 
+  Future<String> _getFileData(String path) async {
+    return await rootBundle.loadString(path);
+  }
+
+  void _setMapStyle(String mapStyle) {
+    setState(() {
+      _nightMode = true;
+      _controller.setMapStyle(mapStyle);
+    });
+  }
+
+  Widget _nightModeToggler() {
+    if (!_isMapCreated) {
+      return null;
+    }
+    return FlatButton(
+      child: Text('${_nightMode ? 'disable' : 'enable'} night mode'),
+      onPressed: () {
+        if (_nightMode) {
+          setState(() {
+            _nightMode = false;
+            _controller.setMapStyle(null);
```

Comment:
Can `_controller` be null here?

Reply:
No, this is only called from `_nightModeToggler` where we first check for `_isMapCreated` which is set alongside with `_controller`.